bigbluebutton-html5/private/config/settings.yml

```diff
@@ -238,6 +239,9 @@ public:
       dataSaving:
         viewParticipantsWebcams: true
         viewScreenshare: true
+      transcription:
+        partialUtterances: true
```

Comment:
Please provide a 1 line comment above explaining what partial utterances are -- for administrators' sake